### PR TITLE
Feat:"added Navbar, Signup and Login page"

### PR DESCRIPTION
**Pull request by sadathshariff**
Have added the Navbar, 
Signup and Login Page for the E-commerce App
E-yard signup

E-yard Login

### CODE REVIEW

File: CSS/auth.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.signup-card {
```
Comment (vaibhav-mahalle) [inline CSS/auth.css]: cart written in text as well as icon side by side. any particular reason for it?

**Review by vaibhav-mahalle — APPROVED** [review]
suggested a minor change please look into it.

File: CSS/auth.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  margin: 6rem auto;
+}
+.signup {
+  max-width: 28rem;
+  margin: auto;
+  padding: 1.5rem;
+  background-color: var(--white-color);
+  box-shadow: rgba(0, 0, 0, 0.07) 0px 1px 1px, rgba(0, 0, 0, 0.07) 0px 2px 2px,
```
Comment (nishasen) [inline CSS/auth.css]: Just a suggestion, give a variable for this.

File: CSS/auth.css (hunk 1 above)
Comment (sadathshariff) [reply CSS/auth.css]: Thanks for pointing it out removing it

**Review by nishasen — COMMENTED** [review]
Just a single comment added, else everything looks good.

**Review by nishasen — APPROVED** [review]

**Pull request merged**